backend/search.py

```diff
@@ -43,3 +44,49 @@ def search():
 
     results_json = results.exclude('image').to_json()
     return results_json
+
+
+@search_blueprint.route("/search_gps", methods=['GET'])
```

Comment:
This belongs in its own file, search_gps.py. Each route/blueprint should be in its own file.

Reply:
Done